src/mlpack/tests/lsh_test.cpp

```diff
@@ -9,103 +9,211 @@
 #include "old_boost_test_definitions.hpp"
 
 #include <mlpack/methods/lsh/lsh_search.hpp>
+#include <mlpack/methods/neighbor_search/neighbor_search.hpp>
 
 using namespace std;
 using namespace mlpack;
 using namespace mlpack::neighbor;
 
+double compute_recall(
+    arma::Mat<size_t> LSHneighbors, 
+    arma::Mat<size_t> groundTruth)
```

Comment:
It's probably better to use const references here: `const arma::Mat<size_t>&` not `arma::Mat<size_t>`.  That will help avoid a copy.